modules/FvwmMFL/FvwmMFL.c

```diff
@@ -246,7 +163,7 @@ static void
 HandleTerminate(int fd, short what, void *arg)
 {
 	fprintf(stderr, "%s: dying...\n", __func__);
-	delete_pid_file();
+	unlink(lockfile);
 	unlink(socket_name);
 	rmdir(socket_basepath);
```

Comment:
Oh, and you probably should not just replace `delete_pid_file()` by `unlink(lockfile)`, but also complement all calls to `unlink(socket_name)` by a call to `unlink(lockfile)` (plus closing the lok file handle)? Or did I miss something here?